fast/stages/2-networking-a-peering/variables.tf

```diff
@@ -167,12 +173,15 @@ variable "psa_ranges" {
   # }
 }
 
-variable "region_trigram" {
-  description = "Short names for GCP regions."
-  type        = map(string)
+variable "regions" {
```

Comment:
Long term I'd rather have regions controlled in a single location, possibly defined in stage 0.

Reply:
maybe... I'm not convinced regions are org-wide vs network-stage specific, let's discuss

Reply:
well... it would make org policies simpler so I see your point

Reply:
let's move this to an issue so we can track it